daprdocs/content/en/reference/cli/dapr-init.md

```diff
@@ -119,4 +119,15 @@ Use the `--set` flag to configure a set of [Helm Chart values](https://github.co
 
 ```bash
 dapr init -k --set global.tag=1.0.0 --set dapr_operator.logLevel=error
+```
+
+You can also specify a private registry to pull container images from. As of now `dapr init -k` does not use specific images for sentry, operator, placement and sidecar. It relies on only Dapr runtime container image `dapr` for all these images.
```

Comment:
```suggestion
You can specify a private registry to pull container images from. 

Note: `dapr init -k` does not use the specific images for Sentry, Operator, Placement and sidecar. It relies on the Dapr runtime container image `dapr` for all these images.
```

Reply:
I presume the "note" above is a bug that will be fixed and then this comment will be removed, yes?

Reply:
@msfussell  Thats true. There is already an issue created for this - https://github.com/dapr/dapr/issues/4712